### ISSUE TITLE
Typescript types not found in v0.1.2

### ISSUE THREAD

**Issue by JumpLink** [codehz/deno-pug#3]
If I upgrade from v0.1.1 to v0.1.2 I cant import `compileFile` and `Options` anymore:

This works with v0.1.1 but not with v0.1.2: 
```ts
import { compileFile, Options as PugOptions } from "pug/mod.ts";
```

Error message:

```bash
TS2305 [ERROR]: Module '"https://deno.land/x/pug@v0.1.2/mod.ts"' has no exported member 'compileFile'.
TS2305 [ERROR]: Module '"https://deno.land/x/pug@v0.1.2/mod.ts"' has no exported member 'Options'.
```

My imports.json:
```json
  "imports": {
    ...
    "pug/": "https://deno.land/x/pug@v0.1.2/",
    ...
  }
```

**Comment by codehz** [comment]
OH, I forgot to update the global.d.ts, fix soon

**Comment by codehz** [comment]
fixed in v0.1.3

**Comment by JumpLink** [comment]
thank you!
